app/src/main/java/io/github/muntashirakon/setedit/utils/SettingsUtils.java

```diff
@@ -30,21 +30,21 @@ public static ActionResult update(@NonNull Context context, @SettingsType String
     public static ActionResult delete(@NonNull Context context, @SettingsType String settingsType,
                                       @NonNull String keyName) {
         if (Boolean.TRUE.equals(Shell.isAppGrantedRoot())) {
-            Shell.Result result = Shell.cmd("settings delete " + settingsType + " " + keyName).exec();
+            Shell.Result result = Shell.cmd(commandBuilder(settingsType) +" delete " + settingsType + " " + keyName).exec();
```

Comment:
This won't work for LOS settings as they will return `null`. You need to find a way to avoid this.